DAO.sol

```diff
@@ -483,8 +488,15 @@ contract DAO is DAOInterface, Token, TokenSale {
     }
 
 
-    function numberOfVotes(uint _proposalID) constant returns (uint _numberOfVotes) {
-        return proposals[_proposalID].votes.length;
+    function isBlocked(address _account) returns (bool){
```

Comment:
From the name, this function sounds as if it were side-effect free, which it is not.